doc/Writerside/topics/starter-topic.md

```
@@ -0,0 +1,12 @@
+# Подсистема интеграции.
```

Markdownlint

Punctuation: '.'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)